docs/dbbackups.md

```diff
@@ -25,3 +23,47 @@ Select a space:
 5. production
 cf logs gsa-fac
 ```
+
+# Media Backups
+```sh
+cf t -s <env>
+```
+
+Bind the backups bucket to the application
+```sh
+cf bind-service gsa-fac backups
+```
+
+Restart the app so changes occur and wait for the instance to come back up
+```sh
+cf restart gsa-fac --strategy rolling
+```
+
+Unbind the existing fac-private-s3 bucket from the app
+```sh
+cf unbind-service gsa-fac fac-private-s3
+```
+
+Rebind the fac-private-s3 bucket with the backups bucket as an additional instance
+```sh
+cf bind-service gsa-fac fac-private-s3 -c '{"additional_instances": ["backups"]}'
```

Comment:
Commenting to note that this is currently blocked, possibly due to an issue with sharing the `backups` instance from prod->staging.